prisms_jobs/jobdb.py

```diff
@@ -206,6 +168,22 @@ def sql_insert_str(job_status):
     return colstr, questionstr, tuple(val)
 
 
+class CompatibilityRow(object):
```

Comment:
I haven't touched the squlite3 module in a while, so this may be a dumb question: why are we using this custom object and now the Row object present in both Python 2 and Python 3? https://docs.python.org/2/library/sqlite3.html#sqlite3.Row and https://docs.python.org/3/library/sqlite3.html#sqlite3.Row both seem to do what you'd want w.r.t. providing dict-like row operation access stuff, and is consistent across versions.

Reply:
Python2 sqlite3.Row.__getitem__ expects a Python2 'str' (bytes), while Python3 sqlite3.Row__getitem__ expects a Python3 'str' (unicode). Since I included the 'unicode_literals' to standardize that string literals are unicode, I needed something to convert unicode -> Python2 'str' when running on Python2. This is something.

Reply:
Ah, fair enough. Again, I haven't played with the squlite3 module in a long time.

To avoid doing a comparison (version-check) during every Row operation, would we be better off using a conditional import or some sort of factory? Even for the n=1000 number of operations Row may do at runtime, I recognize this is a trivial optimization (at best) and not worth more than 1-2 seconds of effort.